packages/sessions-sdk-react/src/components/withdraw-page.tsx

```diff
@@ -92,6 +93,7 @@ const WithdrawForm = ({
           if (result.type === TransactionResultType.Success) {
             toast.success(
               "Successful transferred tokens to your Solana wallet!",
+              <ExplorerLink network={network} txHash={result.signature} />,
```

Comment:
same thing here, what if this doesn't actually succeed on chain? e.g. wh issues/relayer downtime/etc

Reply:
See above, also guarded by `result.type === TransactionResultType.Success`